amnestywebsite/humanity-theme
# Add Alignment Styles To The List Block

Ref: https://github.com/amnestywebsite/humanity-theme/issues/10

**Steps to test**:
1. Add a list block to a post
2. Select the list block and view the toolbar
3. There should now be an option for alignment
4. Switch between the alignments, ensuring the list block changes appropriately 
5. Save and check the same on the front end

Example: http://bigbite.im/v/2XDND4
